### ISSUE TITLE
No notification of new comments (for readers)

### ISSUE THREAD

**Issue by Ultrabenosaurus** [kanadaj/RoyalRoadLegends#45]
As a reader of other people's works, rather than an author looking at their own fictions, the new site seems to contain no system for notifying you when a chapter you have commented on receives new comments. With the old site this was achieved by look at UserCP for subscribed threads with new posts, but since the upgrade neither UserCP nor the new notifications bell icon inform me of new comments. 

As a specific example: [chapter 121 of Binary Soul](http://royalroadl.com/fiction/chapter/96462?page=2#comments).

**Comment by mndauber** [comment]
Bump. I have no way of getting notifications when a new chapter is posted either, which to me is far more pressing. It also wiped my list of subscribed threads and forums, all the books that I have subscribed to email updates in the past. Not sure if that happened to anyone else.

**Comment by kanadaj** [comment]
IF you can give me your userame, I can restore them into follows with email and/or pm notifications

as for comment notifications, I'll look into it.

**Comment by mndauber** [comment]
My username for Royalroad should be Qazicus. Email notifications would be fantastic. Thank you for your quick response.

**Comment by kanadaj** [comment]
Oh, it's you. I'm fairly sure I've already restored it for you a couple hours ago.

**Comment by Gaiadarkstar** [comment]
I went story by story to bookmark those that I had originally had in my subscriptions  I am sure I missed quite a few I don't mind the new system for that so much but no notifications for new comments are way too annoying are we supposed to keep each page bookmarked that we comment on to see if someone responds.  There needs to be notifications for responses somehow soon I want to know what people say to what I have posted I am sure I have already missed many responses to questions I have asked in other posts.  This is the only big problem other than losing some of the stories I was following, not sure which ones I might have missed that I really liked.
